codegen/package_test.go

```diff
@@ -131,5 +131,5 @@ func TestGoCustomTypeError(t *testing.T) {
 	spec := &compile.StructSpec{}
 	_, err := h.TypeFullName(spec)
 	assert.Error(t, err)
-	assert.Equal(t, "goCustomType called with native type (*compile.StructSpec) &{false   0 [] map[]}", err.Error())
+	assert.Equal(t, "goCustomType called with native type (*compile.StructSpec) &{false   0 []  map[]}", err.Error())
```

Comment:
why the extra space?

Reply:
I don't know actually!  Are the recent commits causing this?

Reply:
Formatting the error with `%+v` revealed that the 2 extra spaces between `false` and `0` are for `Name` and `File` empty strings, the 1 extra space between `[]` and `map[]` is for the `Doc` empty string:

```
expected: "goCustomType called with native type (*compile.StructSpec) &{false   0 []  map[]}"
actual: "goCustomType called with native type (*compile.StructSpec) &{linkOnce:false Name: File: Type:0 Fields:[] Doc: Annotations:map[]}"
```